test/e2e/framework/ingress_utils.go

```diff
@@ -997,6 +1104,26 @@ func (j *IngressTestJig) ConstructFirewallForIngress(gceController *GCEIngressCo
 	return &fw
 }
 
+func (j *IngressTestJig) DetectDistinctResponse() (int, error) {
```

Comment:
What does DistinctResponse mean in this context? Also, why not return the actual set of responses? Returning the number of responses seems to be unfriendly for debugging.

Reply:
ack

Reply:
I will rename it to `GetDistinctReponseFromIngress`